.github/workflows/revert-tracker.yml

```diff
@@ -0,0 +1,46 @@
+name: revert printer
+
+on:
+  repository_dispatch:
+  push:
+  schedule:
+    # At 16:39 (9:39 AM PST) on Monday
+    - cron: 39 16 * * 1
+
+jobs:
+  revert_printer:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          path: test-infra
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          repository: pytorch/pytorch
+          path: pytorch
+          fetch-depth: 0
+      - run: |
+          cd test-infra
+          python3 -m pip install rockset==0.8.3
+          python3 -m torchci.scripts.reverts > reverts.md
```

Comment:
Could we put this in a subfolder and add a datestamp to this file so that it doesn't get overwritten every week?

Reply:
made it go to a file thats just titled with the date of the monday